test/sandbox/app/controllers/layouts_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Tests for render with/without layouts
+class LayoutsController < ActionController::Base
+  layout "global_for_action", only: :global_for_action
```

Comment:
For some reason `"application"` layout is not searched... maybe this is new in rails, and we need an ApplicationController?

Reply:
You were right :) Adding an `ApplicationController` makes the tests pass!

See https://api.rubyonrails.org/classes/ActionView/Layouts.html#module-ActionView::Layouts-label-Layout+assignment